pipeline/minimmit.md

```diff
@@ -56,134 +57,142 @@ fn verify(c, c') -> bool;
 
 ## 7. Helpers
 
+In every helper function, the first parameter `r` is the replica whose local state the helper inspects or updates .
```

Comment:
Two nits:

* Extra space between `updates` and `.`.
* For consistency, this should be in italic and there should be a `\n`.

Reply:
Fixed in https://github.com/commonwarexyz/monorepo/pull/1151/commits/e08ddd4c3922b11a152d3d75aa94aa21fdef5901